examples/angular_bazel_architect/BUILD.bazel

```diff
@@ -112,6 +112,36 @@ architect_test(
     ],
 )
 
+architect(
+    name = "serve",
```

Comment:
does it work? last we discussed it wasn't picking up changes right?

Reply:
Ah yeah so it working.
It wasn't working last time since my VM was out of file watchers.
I guess webpack dosen't have a fallback to stating files